File: src/main/java/com/anthonymariotti/ticketmaster/TicketMasterServer.java
package com.anthonymariotti.ticketmaster;

import com.anthonymariotti.ticketmaster.utilities.TicketMasterLogger;
import net.fabricmc.api.DedicatedServerModInitializer;

public class TicketMasterServer extends TicketMasterCore implements DedicatedServerModInitializer {

    @Override
    public void onInitializeServer() {
        TicketMasterLogger.info("Initializing Ticket Master");
        super.init();
    }
}

File: src/main/kotlin/com/anthonymariotti/ticketmaster/TicketMasterServer.kt
package com.anthonymariotti.ticketmaster

import com.anthonymariotti.ticketmaster.core.TicketMasterCore
import net.fabricmc.api.DedicatedServerModInitializer

object TicketMasterServer : TicketMasterCore(), DedicatedServerModInitializer {

    override fun onInitializeServer() {
        initCore()


    }

}